css/trackers.scss

```diff
@@ -121,13 +122,18 @@ body {
 ul.js-menu-item-list {
     list-style-type: none;
     padding-left:14px;
+    margin-top:0;
 }
 
 // .js-site-item
 ul.js-menu-item-list li {
     padding-top:17px;
 }
 
+ul.js-menu-item-list li.js-site-co:first-child {
+    padding-top:0 !important;
```

Comment:
Yeah. I can fix this. I think we should look at refactoring the css/scss next week.